github-dark.css

```diff
@@ -3828,7 +3828,7 @@
   .zhc-pr-issue-connector__info, .zhu-tag-default, .zhc-chart-container__header,
   .zhc-issues-list__header, .zhc-toggleable--is-disabled,
   .zhc-item-table__row:hover, .zh-collapse-control__main-icon,
-  .zhc-sidebar-profile {
+  .zhc-sidebar-profile, .tag-input .suggester {
```

Comment:
Please move this out of the ZenHub (`.zhc`) section and up to around line 1913 where the background color is `#181818` and border is `#343434` or `#484848` whichever looks better.

Reply:
@Mottie done and done